nipype/interfaces/dcm2nii.py

```diff
@@ -306,27 +355,30 @@ class Dcm2niix(CommandLine):
 
     >>> from nipype.interfaces.dcm2nii import Dcm2niix
     >>> converter = Dcm2niix()
-    >>> converter.inputs.source_names = ['functional_1.dcm', 'functional_2.dcm']
-    >>> converter.inputs.compress = 'i'
-    >>> converter.inputs.single_file = True
-    >>> converter.inputs.output_dir = '.'
-    >>> converter.cmdline # doctest: +SKIP
-    'dcm2niix -b y -z i -x n -t n -m n -f %t%p -o . -s y -v n functional_1.dcm'
+    >>> converter.inputs.source_dir = 'dicomdir'
+    >>> converter.inputs.compression = 5
+    >>> converter.inputs.output_dir = 'ds005'
+    >>> converter.cmdline
+    'dcm2niix -b y -z y -5 -x n -t n -m n -o ds005 -s n -v n dicomdir'
 
     >>> flags = '-'.join([val.strip() + ' ' for val in sorted(' '.join(converter.cmdline.split()[1:-1]).split('-'))])
     >>> flags
-    ' -b y -f %t%p -m n -o . -s y -t n -v n -x n -z i '
+    ' -5 -b y -m n -o ds005 -s n -t n -v n -x n -z y '
```

Comment:
I'm fine to remove this as well, just kept around since it previously existed

Reply:
Looks like @satra specifically added this in #1529. Guessing the flag ordering was non-deterministic, since he also added `#doctest: +SKIP` to the `cmdline` line. If the `cmdline` check is working consistently, I'd say go ahead and drop.